temporian/core/evaluation.py

```diff
@@ -258,6 +258,59 @@ def build_schedule(
     return schedule
 
 
+def has_leak(
+    output: EvaluationQuery,
+    input: Optional[EvaluationQuery] = None,
+) -> bool:
+    """Tests if a node depends on a leak operator.
+
+    Tests if a [`Node`][temporian.Node] or collection of nodes depends on the
+    only operator that can introduce a future leakage:
+    [`tp.leak()`][temporian.leak].
+
+    Single input output example:
+        ```python
+        >>> a = tp.input_node([("f", float)])
+        >>> b = tp.moving_sum(a, 5)
+        >>> c = tp.leak(b, 6)
+        >>> d = tp.prefix("my_prefix_", c)
+        >>> e = tp.moving_sum(d, 7)
+        >>> # The computation of "e" contains a leak.
+        >>> assert tp.has_leak(e)
```

Comment:
I think it would read better in the examples to print the `True` or `False` results instead of assert them

Reply:
Good point.
*Use returned value